src/main/java/com/danielagapov/spawn/Repositories/IFriendTagRepository.java

```diff
@@ -3,10 +3,14 @@
 import com.danielagapov.spawn.Models.FriendTag;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
+import org.springframework.data.jpa.repository.Query;
 
 import java.util.UUID;
+import java.util.List;
 
 @Repository
 public interface IFriendTagRepository extends JpaRepository<FriendTag, UUID> {
     // The JpaRepository interface already includes methods like save() and findById()
+    @Query(value = "SELECT * FROM id WHERE owner_id = ?1", nativeQuery = true)
+    List<UUID> findTagsByOwner(UUID ownerId);
```

Comment:
Change to: `List<FriendTag> findByOwnerId(UUID ownerId);` for JPA naming conventions.

Reply:
Also, notice how I changed the return type to be `FriendTag`s instead of `UUID`s, because that's what that SQL is returning. If you mean to return ids then adjust the SQL to not return the row, but that id property you're looking for